docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -307,10 +307,14 @@ Since release 4.3.2 you can use the `search.reopt` query optimizer hint (shown a
 
 ### 3.4.2 Properties-based datastore configuration
 
-Normally, a Liberty application that uses one or more Derby or Db2 datastores will require a datasource to be defined within the Liberty server.xml file for each database. One drawback to this approach is that each of the datasources are statically defined in the 'server.xml' file, which means that any updates (modifications, additions, etc.) will require a server re-start.
+Normally, a Liberty application that uses one or more JDBC datastores will require a datasource to be defined within the Liberty server.xml file for each database. One drawback to this approach is that each of the datasources are statically defined in the 'server.xml' file, which means that any updates (modifications, additions, etc.) will require a server re-start.
 
 As part of it's multi-tenant support, the FHIR server provides an alternate mechanism which consists of a single “proxy datasource” along with a set of properties configured in the `fhir-server-config.json` file. This proxy datasource can be used by the JDBC persistence layer implementation to establish connections to either Derby or Db2 databases. This approach allows for new datastores to be configured without the need to restart the FHIR server.
 
+Since release 4.5.0, the FHIR server supports standard JDBC datasources defined in the server '.xml' files. Datasource elements can be defined in the 'server.xml' file but can also be defined in the 'configDropins/overrides' directory to simplify configuration, especially where automation may need to add multiple datasources.
```

Comment:
```suggestion
Since release 4.5.0, the IBM FHIR Server supports standard JDBC datasources defined in the server '.xml' files. Datasource elements can be defined in the 'server.xml' file but can also be defined in the 'configDropins/overrides' directory to simplify configuration, especially where automation may need to add multiple datasources.
```
A few points... 

1 - So when you update this guide it goes live right away, we should probably use 4.4.1 as a reference rather than the non-existent 4.5.0
2 - I don't think we should tell folks to update the server.xml directly, perhaps invert the recommendation and use the but to updates to the server.xml.

Reply:
The problem here is describing functionality present in a release that's not out yet, but the code is in master. I think it's preferable to say when this function is available, so people don't expect it to be behavior of a current release.

Reply:
I've reworded so we don't suggest adding the datasource to server.xml